docs/conf.py

```diff
@@ -70,6 +70,8 @@
 # for a list of supported languages.
 language = 'en'
 
+gettext_compact = False
```

Comment:
This causes a `.pot` file to be generated per `.rst` file. Otherwise, there’s 1 `.pot` file per root documentation folder or `.rst` file (e.g. one for `intro`, one for `topics`).

I’m unsure about my choice here. I used it to make it easier for me to translate only the tutorial, but it may be better to leave the default value here and have fewer files.